feature/google/src/test/java/com/revenuecat/purchases/google/BillingWrapperTest.kt

```diff
@@ -315,6 +318,80 @@ class BillingWrapperTest {
         )
     }
 
+    @Test
+    fun `obfuscatedAccountId is set for non-transfer purchases`() {
+        setup()
+        val appUserID = "jerry"
+
+        val skuDetails = stubSkuDetails(productId = "product_a")
+        val activity: Activity = mockk()
+
+        mockkStatic(BillingFlowParams::class)
+        val mockBuilder = mockk<BillingFlowParams.Builder>(relaxed = true)
+        every {
+            BillingFlowParams.newBuilder()
+        } returns mockBuilder
+
+        every {
+            mockBuilder.setSkuDetails(any())
+        } returns mockBuilder
+
+        every {
+            mockClient.launchBillingFlow(any(), any())
```

Comment:
we had mocked the params here, but i didn't think that was necessary anymore and removed them...